### PR TITLE
feat: Add support for mark_unread msg or thread

### PR DESCRIPTION
**Pull request by guerinoni**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

## Description of the pull request

Add new api `mark_unread` for marking thread or message (`thread_id` // `message_id`)

### CODE REVIEW

File: stream_chat/channel.py [hunk 1]
```diff
@@ -135,6 +135,10 @@ def mark_read(self, user_id: str, **data: Any) -> StreamResponse:
         payload = add_user_id(data, user_id)
         return self.client.post(f"{self.url}/read", data=payload)
 
+    def mark_unread(self, user_id: str, **data: Any) -> StreamResponse:
+        payload = add_user_id(data, user_id)
+        return self.client.post(f"{self.url}/read", data=payload)
```
Comment (gumuz) [inline stream_chat/channel.py:140]: this url seems incorrect (read vs unread)
are you sure the base of the url is correct too?

**Review by gumuz — CHANGES_REQUESTED** [review]

File: stream_chat/channel.py (hunk 1 above)
Comment (guerinoni) [reply stream_chat/channel.py]: Thanks :) copy&paste error from the above method, fixed

**Review by gumuz — COMMENTED** [review]
don't know enough about these endpoints to test their correctness, but tests are failing

**Review by gumuz — APPROVED** [review]
LGTM

**Pull request merged**